src/lib/generics-events-subscribers/erc20-event-subscriber-factory.ts

```diff
@@ -0,0 +1,18 @@
+import { IDexHelper } from '../../dex-helper';
+import { Address } from '../../types';
+import { ERC20EventSubscriber } from './erc20-event-subscriber';
+
+const subscriberMap: Record<Address, ERC20EventSubscriber> = {};
```

Comment:
I don't understand why wee need this mapping? Just to keep instance running after it was returned?

Reply:
If some others dexes want to the same erc20 tracking